api/openapi/v1/support-scheduler.yaml

```diff
@@ -329,10 +316,13 @@ paths:
           description: for unknown or unanticipated issues
   /v1/intervalaction/{id}:
     get:
-      description: Fetch a specific IntervalAction by database generated id. May return
-        null if no interval action with the id is found. Returns Internal Service
-        Error (HTTP 500) for unknown or unanticipated issues. NotFoundException (HTTP
-        404) if no IntervalAction is found with the provided id.
+      description: Fetch a specific set of interval events matching given name
```

Comment:
Got YAML validation failure in this description section.

Reply:
@cherrycl Good catch. Fixed it, and will make it a point to run my `YAML` validator going forward for this and other `Swagger` docs 👍